manifests/g/Gloo/cli/0.1.6/Gloo.cli.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Gloo.cli
+PackageVersion: 0.1.6
+InstallerType: portable
+Commands:
+- gloo
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
`x86` validations are performed on a 32-bit OS. This package installs successfully in a 64-bit VM.